news/2022-07-26-project-loved-july-2022.md

```diff
@@ -0,0 +1,385 @@
+---
+layout: post
+title: "Project Loved: July 2022"
+date: 2022-07-26 07:00:00 +0000
+---
+
+Check out the latest picks from the captains at Project Loved!
+
+![](/wiki/shared/news/banners/project-loved-2.jpg)
+
+For those unfamiliar, Project Loved hosts monthly votings to decide which nominated maps get added to the [Loved category](https://osu.ppy.sh/beatmapsets?s=loved). You can click on each of the map's banners in this post to go to its poll on the forums, or visit [the forum](https://osu.ppy.sh/community/forums/120) for a list of all of the polls. Polls will be open for 10 days, so make sure to cast your votes in time! More info can be found on the [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh)!
+
+Detailed results for the previous round can be found on the [poll results page of the website](https://loved.sh/statistics) or in the following forum posts:
+
+- [osu!](https://osu.ppy.sh/community/forums/posts/8694723)
+- [osu!taiko](https://osu.ppy.sh/community/forums/posts/8694722)
+- [osu!catch](https://osu.ppy.sh/community/forums/posts/8694721)
+- [osu!mania](https://osu.ppy.sh/community/forums/posts/8694720)
+
+Before getting into this month's picks, we want to give thanks to [Pachiru](https://osu.ppy.sh/users/2850983) for his effort in coordinating Project Loved throughout 2021 and 2022 — there were many rounds where this whole shebang would have ground to a halt if he hadn't been there to maintain the project, contact mappers, keep everything on-schedule, and make these announcements. He recently stopped working on Loved, but in all likelihood he'll still be a familiar face around mapping, modding, and moderation.
+
+In other news, the osu! captains of Project Loved have some important updates to share:
+
+First, they've published an **application to join the osu! captains**, available at [**this Google form**](https://docs.google.com/forms/d/e/1FAIpQLSd9jLuzv86yLEnx1MQYzdoLqMccahspgAzZSEn4n1Qsr-DLHQ/viewform). Becoming a captain never was supposed to be a closed-off process, but it may have felt this way with the lack of explicit outreach. We hope to see some more friends inspired to help out with picking osu!'s Loved candidates!
+
+Second, and also related to making osu!'s Project Loved section feel more accessible, **a [Discord server](https://discord.gg/gn58Uk5sTE) was opened** as another place to give feedback or just hang out and chat about all things osu! Loved. As a reminder, Project Loved has presence in many other Discord servers that you can find linked on [the website](https://loved.sh/more).
```

Comment:
wat

Reply:
remember that "osu!" can either mean the mode or the game, and the latter interpretation would lead to a wonky sentence. having it this way is cleaner and still semantically correct.

also it'd be "Loved section" and not "Project Loved section" anyway

Reply:
i think u made a typo or something in ur suggestion o-o

Reply:
if anything you'd swap this out for ```making the Project Loved section feel more accessible``` or ```making Project Loved feel more accessible``` rather than applying this one, which is i assume you what you were going for

Reply:
did latter, originally i was trying to say the osu standard part of loved, but it also works to say it's more accessible overall.